runtime/commonMain/src/kotlinx/serialization/internal/Enums.kt

```diff
@@ -44,6 +45,7 @@ internal class EnumDescriptor @JvmOverloads constructor(
     }
 }
 
+@InternalSerializationApi
 open class CommonEnumSerializer<T>(serialName: String, val values: Array<T>, valuesNames: Array<String>) :
     KSerializer<T> {
```

Comment:
Codestyle, should be 
```
@InternalSerializationApi
open class CommonEnumSerializer<T>(
    serialName: String,
    val values: Array<T>,
    valuesNames: Array<String>
) : KSerializer<T> {
    
    override val descriptor: SerialDescriptor = EnumDescriptor(serialName, valuesNames)
```

Reply:
Any tips to set up the IDEA formatter properly? Standard Kotlin code style does not address this.